kolibri/plugins/coach/assets/src/views/common/QuizStatus.vue

```diff
@@ -100,8 +100,10 @@
           :layout8="{ span: 4 }"
           :layout12="{ span: 2 }"
         >
+          <span id="toggle-quiz-visibility-label">Toggle quiz visibility</span>
           <KSwitch
             name="toggle-quiz-visibility"
+            :aria-labelledby="'toggle-quiz-visibility'"
```

Comment:
Should this line match the `id='toggle-quiz-visibility-label'` in the span above?

Reply:
Also, shouldn't the prop be named `ariaLabelledBy` rather than `:aria-labelledby`?